### DISCUSSION TITLE
New Game Mode: Inverted Royale

### DISCUSSION THREAD

**Discussion by md-hexdrive** [BattlesnakeOfficial/feedback#154] [General]
TLDR: Behaves similar to the regular Royale game mode, but instead of hazards spawning from the outside in; they spawn starting from the inside (the center of the game board) and move outwards toward the edges. This would throw off snakes that normally try to control the middle of the board and force opponents into the game board edges or "sauce" in Royale and Standard game modes.

Not to much has to be changed to implement this compared to the normal Royale game mode to implement this. The ruleset could be implemented as a modification of the Royale ruleset, with just the hazard spawn rules replaced.

**Comment by md-hexdrive** [comment]
In this game mode, the first hazards would spawn like this:
![image](https://user-images.githubusercontent.com/57779180/151451748-28f28612-4962-4ba4-8769-8fa654aa0821.png)

Over time, they grow:
![image](https://user-images.githubusercontent.com/57779180/151451577-f875ede4-79b0-4b00-bb47-62a3701d5332.png)
![image](https://user-images.githubusercontent.com/57779180/151451832-22c52b6e-3486-470f-9a8f-8bab0fb0d060.png)
![image](https://user-images.githubusercontent.com/57779180/151451900-4d61bdca-5702-4a3d-9379-6df8b84c31f3.png)
